doc/nrf/releases/release-notes-latest.rst

```diff
@@ -91,6 +91,77 @@ The following list summarizes the most important changes inherited from upstream
 
   * Removed the ``net_idx`` parameter from the Health Client model APIs because it can be derived (by the stack) from the ``app_idx`` parameter.
 
+* Drivers:
+
+  * Clock control:
+
+    * Fixed an issue in the nRF clock control driver that could lead to a fatal error during the system initialization, when calibration was started before kernel services became available.
+
+  * Display:
+
+    * Add support for temperature sensors in the ssd16xx driver.
+
+  * Entropy:
+
+    * Fixed a race condition in the entropy_nrf5 driver that could result in a miss of the wake up event (and caused the ``kernel.memory_protection.stack_random`` test to fail).
+
+  * Flash:
+
+    * Extended the flash API with the ``flash_get_parameters()`` function.
+    * Fixed an issue in the soc_flash_nrf driver that caused operations to fail on Bluetooth central that had multiple connections.
+    * Added support for 2 IO pin setup in the nrf_qspi_nor driver.
+    * Added support for sub-word lengths of read and write transfers in the nrf_qspi_nor driver.
+    * Improved handling of erase operations in the nrf_qspi_nor, spi_flash_at45, and spi_nor drivers. Now the operation is not started if it cannot be successfully completed.
+
+  * GPIO:
+
+    * Removed deprecated API functions and macros.
+    * Improved allocation of GPIOTE channels in the gpio_nrfx driver.
+
+  * I2C:
+
+    * Fixed handling of scattered transactions in the i2c_nrfx_twim driver by introducing an optional concatenation buffer.
+    * Used a time limit (100 ms) when waiting for transaction completion, in both nRF drivers.
+
+  * LoRa:
+
+    * Added support for SX126x transceivers.
+
+  * PWM:
+
+    * Clarifed expected API behavior regarding zero pulse length and non-zero pulse equal to period length.
+
+  * Sensors:
+
+    * Added support for the IIS2DH accelerometer.
+    * Added the ``sensor_attr_get()`` API function for getting the value of a sensor attribute.
+
+  * Serial:
+
+    * Clarified in the UART API that the ``UART_RX_RDY`` event is to be generated before ``UART_RX_DISABLED`` if any received data remain. Updated all drivers in this regard.
+    * Changed uart_nrfx_uart/uarte drivers to use the DT ``hw-flow-control`` property instead of Kconfig options.
+    * Fixed disabling of the TX interrupt in the uart_nrfx_uart driver.
+    * Fixed the uart_nrfx_uarte driver to prevent spurious ``UART_RX_BUF_REQUEST`` events.
+
+  * SPI:
+
+    * Updated ``cs-gpios`` properties in DT SPI nodes with proper GPIO flags and updated SPI drivers to use them.
```

Comment:
It's about changes done in https://github.com/zephyrproject-rtos/zephyr/pull/26684. `cs-gpios` properties in devicetree nodes are updated with proper flags defined in the GPIO driver API (like [here](https://github.com/zephyrproject-rtos/zephyr/pull/26684/files#diff-7336bec34627848302977a5aeb8ec402L50) from `0` to `GPIO_ACTIVE_LOW`) and implementations of related drivers (actually, those are not SPI drivers as I initially wrote, I need to correct this) are updated to use those values from devicetree instead of some hard-coded or assumed ones when configuring CS lines.
I think it should be rather clear to developers.

Reply:
So far I changed "SPI drivers" to "related drivers". Any other changes to apply here?

Reply:
How about:
```suggestion
    * Updated the ``cs-gpios`` properties in DT SPI nodes to use values from GPIO flags instead of hard-coded values.
      Updated the related drivers to use the ``cs-gpios`` properties.
```
Is the last line correct, or what else does "them" refer to?

Reply:
The `cs-gpio` property consist of three components: a gpio device, a pin within this device, and finally flags specifying how to configure this pin. The drivers that were updated in the PR mentioned above used so far only the first two of those components. Now they use also the flags. But first those flags in the `cs-gpio` properties had to be updated with proper values (0 was not correct, as it would mean `GPIO_ACTIVE_HIGH`, while in most cases CS lines are active low).
Thus, I propose to put it this way:
```
    * Updated the ``cs-gpios`` properties in DT SPI nodes with proper GPIO flags specifying the active level.
      Updated the related drivers to use the flags from ``cs-gpios`` properties instead of hard-coded values.
```

Reply:
👍 Looks good, go for that. :)

Reply:
I just did. Thanks!